.eslintrc.json

```diff
@@ -0,0 +1,115 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true
+  },
+  "extends": [
+    "eslint:recommended",
+    "plugin:react/recommended",
+    "plugin:react/jsx-runtime",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:@typescript-eslint/recommended-requiring-type-checking",
+    "plugin:prettier/recommended",
+    "plugin:import/recommended"
+  ],
+  // Specifying Parser
+  "parser": "@typescript-eslint/parser",
+  // Configuring third-party plugins
+  "plugins": ["react", "react-hooks", "@typescript-eslint"],
+  // Resolve imports
+  "settings": {
+    "import/resolver": {
+      "typescript": {
+        "project": "./tsconfig.json"
+      }
+    },
+    "react": {
+      "version": "18.x"
+    }
+  },
+  "rules": {
+    "linebreak-style": "off",
+    "radix": "error",
+    "no-new-wrappers": "error",
+    "curly": "error",
+    "no-mixed-operators": "warn",
+    // Configure prettier
+    "prettier/prettier": [
```

Comment:
Add the prettier rules in a specific file

Reply:
done